API.md

```diff
@@ -23,58 +24,386 @@ Response example
+
+Args:
+    count(string): Number of items
+
+Returns:
+    dict: of all added items
+
+
+### get_server_friendly_name
+
+
+### get_server_id
+
+
+### get_server_list
+Find all servers published on plextv
+
+
+### get_server_pref
+Return a specified server preference.
+
+Args:
+    pref(string): 'name of preference'
+
+Returns:
+    String: ''
+
+
+### get_server_prefs
+
+
+### get_servers
+All servers
+
+Returns:
+        json:
+            ```
+            {"MediaContainer": {"@size": "1", "Server":
+                {"@name": "dude-PC",
+                "@host": "10.0.0.97",
+                "@address": "10.0.0.97",
+                "@port": "32400",
+                "@machineIdentifier": "1234",
+                "@version": "0.9.15.2.1663-7efd046"}}}
+            ```
+
+
+### get_servers_info
+Graps info about the server
+
+Returns:
+        json:
+            ```
+            [{"port": "32400",
+              "host": "10.0.0.97",
+              "version": "0.9.15.2.1663-7efd046",
+              "name": "dude-PC",
+              "machine_identifier": "1234"
+              }
+            ]
+            ```
+
+
+### get_sessions
+
+
+### get_settings
+Fetches all settings from the config file
+
+Args:
+    key(string, optional): 'Run the it without args to see all settings'
+
+Returns:
+        json:
+            ```
+            {General: {api_enabled: true, ...}
+             Advanced: {cache_sizemb: "32", ...}}
+            ```
+
+
+### get_stream_type_by_top_10_platforms
+
+
+### get_stream_type_by_top_10_users
+
+
+### get_sync_item
+Return sync item details.
+
+Args:
+    sync_id(string): unique sync id for item
+    output_format(string, optional): 'xml/json'
+
+Returns:
+    List:
+        ```
+        {"data": [
+                    {"username": "username",
+                     "item_downloaded_percent_complete": 100,
+                     "user_id": "134",
+                     "failure": "",
+                     "title": "Some Movie",
+                     "total_size": "747195119",
+                     "root_title": "Movies",
+                     "music_bitrate": "128",
+                     "photo_quality": "49",
+                     "friendly_name": "username",
+                     "device_name": "Username iPad",
+                     "platform": "iOS",
+                     "state": "complete",
+                     "item_downloaded_count": "1",
+                     "content_type": "video",
+                     "metadata_type": "movie",
+                     "video_quality": "49",
+                     "item_count": "1",
+                     "rating_key": "59207",
+                     "item_complete_count": "1",
+                     "sync_id": "1234"}
+                ]
+        }
+        ```
+
+
+### get_sync_lists
+
+
+### get_sync_transcode_queue
+
+
+### get_user_details
+Get all details about a user from plextv
+
+
+### get_user_list
+
+
+### notify
+
+
+### random_arnold_quotes
```

Comment:
Well when all you need to do is slap decorator to add it to the api this is what happens... :P, while that being said, some stuff should be imported manually. I think i went a little overboard..